25.md

```diff
@@ -52,6 +52,22 @@ func make_like_event(pubkey: String, privkey: String, liked: NostrEvent) -> Nost
 }
 ```
 
+Reactions to a website
+---------------------
+
+If the target of the reaction is a website, the reaction SHOULD be a `kind 17` event and SHOULD include an `r` tag with the website's URL.
+
+```json
+{
+  "kind": 17,
+  "content": "⭐",
+  "tags": [
+    ["r", "https://example.com/"]
```

Comment:
Normalize the URL, like on https://datatracker.ietf.org/doc/html/rfc3986#section-6. 

You might also want to add guidance on fragments (can you like a section of a page?)

Also, keep in mind that there is no `startsWith` filter in Nostr, which means that if you want to get all reactions to a domain name (or path), you have to create individual tags for each.

Reply:
Can we add this? And some examples maybe?